### ISSUE TITLE
Remove the right-side panel

### ISSUE THREAD

**Issue by Monniasza** [MultiMachineBuilder/MultiMachineBuilder#129]
The right side panel is used rarely in the game.
Its upper half is used only by the chatter and its lower half is unused.
Additionally, completely remove any item entry code which uses the lower right panel,
and refactor the chatter and its GUI to use fullscreen tabs.

**Comment by mark-jardine** [comment]
Could I contribute to this?

**Comment by Monniasza** [comment]
Yes. I'll assign you.

**Comment by Monniasza** [comment]
@mark-jardine Follow the [contributing guidelines](https://github.com/MultiMachineBuilder/MultiMachineBuilder/blob/master/CONTRIBUTING.md) for guidance.
You can also learn at https://github.com/firstcontributions/first-contributions

**Comment by mark-jardine** [comment]
Thanks I'll have a look

**Comment by mark-jardine** [comment]
@Monniasza 

Hi I was wondering, is it necessary to use Eclipse?

**Comment by Monniasza** [comment]
Eclipse is needed to have all project settings work and required plugins installed. The project will compile without it, but you will get unnecessary warnings and setup will be more complex.

**Comment by mark-jardine** [comment]
Hi, I have moved over to Eclipse, but I am having some issues with the plugin- [Eclipse Maven integration (M2E extension) for null analysis](https://github.com/lastnpe/eclipse-external-annotations-m2e-plugin). 

I get the following error: 
`An error occurred while collecting items to be installed
session context was:(profile=C__Users_markj_eclipse_java-2021-09_eclipse, phase=org.eclipse.equinox.internal.p2.engine.phases.Collect, operand=, action=).
Result of processing steps.
OK
OK
OK
Public key not found for -9147676190077153272.
OK
Result of processing steps.
OK
OK
OK
Public key not found for -9147676190077153272.
OK`

Do you know if there is a workaround for this? Apologies but this is my first time contributing and I would be interested in contributing to your project. 
![eclipse-error](https://user-images.githubusercontent.com/77744624/234613425-cc00825b-9d44-42cd-8173-d6273bfb8a60.png)

**Comment by Monniasza** [comment]
@mark-jardine Follow the guide in the pl ugin you have problems with, don't blindly copy the link into Eclipse.

**Comment by Monniasza** [comment]
@mark-jardine To get appropriate coding guidelines:
1. Log in to SonarCloud with GitHub,
2. Create a SonarCloud access token and copy it
3. Connect the SonarLint to SonarCloud using the copied token
4. Connect your MultiMachineBuilder project to the `MultiMachineBuilder_MultiMachineBuilder` project in `MultiMachineBuilder` organization in SonarLint

**Comment by mark-jardine** [comment]
@Monniasza Thanks, are the rules those that are outlined in Quality Profiles/Java?

**Comment by Monniasza** [comment]
@mark-jardine Yes, they are outlined as `MultiMachineBuilder` Java quality profile

**Comment by Monniasza** [comment]
@mark-jardine If you don't respond by May 31, I'll take over this task.

**Comment by dlewburg** [comment]
@Monniasza do you still need help with the tasks labeled help wanted?

**Comment by dlewburg** [comment]
@Monniasza I meant labeled good first issue

**Comment by Monniasza** [comment]
@dlewburg Yes

**Comment by Monniasza** [comment]
@dlewburg Are you still working on it? If not, I can propose easier tasks and take over it.

**Comment by Monniasza** [comment]
@dlewburg If you don't respond by July 5, I'll take over this task

**Comment by LenkaBuebnkova** [comment]
Hi, is this task still available?

**Comment by PKCPablo** [comment]
@Monniasza, I can give it a try, yup. When I delete the right-side panel, where should I put the chat??

**Comment by Monniasza** [comment]
@PKCPablo You should use regular tabs for the chat editor.

**Comment by PKCPablo** [comment]
@Monniasza what do you mean by regular tabs?

**Comment by Monniasza** [comment]
@PKCPablo These are used by most machines. For example visit the speaker block.

**Comment by PKCPablo** [comment]
@Monniasza Right now I won't have that much free time due to the holidays, but after the holidays, I would like to keep contributing. I also have a friend who would be interested in contributing, do you have more **good first issues** to collaborate for him?

**Comment by Monniasza** [comment]
@PKCPablo Not now, but later they might appear, so subscribe to issues.

**Comment by Monniasza** [comment]
@Hackerfiera Will you have time after New Year Eve?
HR SRU TAOEUPL AF KPA*L TPHU KPA*L KWRAOER AOEF H-F

**Comment by Hackerfiera** [comment]
YES of course, but I want to know exactly what you want, completely eliminate the right part (the entire panel) and have the chat be in another panel that you can also maximize?

**Comment by Hackerfiera** [comment]
ok perfect, let's see if tomorrow/the day after I can start looking at it

**Comment by Monniasza** [comment]
@Hackerfiera The class to refactor is mmb.menu.world.window.TabWorld.

**Comment by Hackerfiera** [comment]
I understand, the part on the right removed, I think I know how to do it, I will change some class and when I have it I will make a PR.
Do you want the 'dialog' theme in another window next to 'Recipes' as seen in the image? I just don't quite understand what you were referring to.
![image](https://github.com/MultiMachineBuilder/MultiMachineBuilder/assets/79371273/85239b14-3824-4430-8a6e-01e22e1241c5)

**Comment by Monniasza** [comment]
@Hackerfiera Yes, the tab should appear right to the existing tabs, like all other blocks do. The windows should also show by default.

**Comment by Hackerfiera** [comment]
OK, I have removed the panel on the right as you can see and added the dialogue tab.
![image](https://github.com/MultiMachineBuilder/MultiMachineBuilder/assets/79371273/19833dbf-5757-432a-a455-743ddec0d267)
Literally I have put what was in the old dialog to this one, moreover when you click on the tab you can see this, I don't know if this is what you mean, now I make a pull request so you can accept the changes or tell me if I am missing something.
![image](https://github.com/MultiMachineBuilder/MultiMachineBuilder/assets/79371273/dd02186e-99e5-4337-a86b-3d5d27f63b81)

**Comment by Monniasza** [comment]
@Hackerfiera You should replace all dialog creations with tab creations, without a dialog tab which contains dialogs.
